packages/website/lib/api.js

```diff
@@ -267,3 +274,26 @@ export async function listPins(status, token) {
 
   return res.json();
 }
+
+/**
+ * Deletes a pin request.
+ *
+ * @param {string} requestid
+ */
+export async function deletePinRequest(requestid) {
+  const tokens = await getTokens();
+  if (!tokens[0]) {
```

Comment:
A bit confusing; could we use `tokens.length > 0`?

Reply:
Possibly, but with this we're checking if `tokens[0]` is truthy, as it could be null. 
Tokens could be `[null, null]` and .length would be 2